### PR TITLE
Update MAINTAINERS.md

### PR DESCRIPTION
**Pull request by ShuzZzle**
**What this PR does**:
Update MAINTAINERS
Drop Ken
Add myself
Add Tom Hayward ( he will do that himself )

**Which issue(s) this PR fixes**:
None

**Checklist**
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 - Granville Schmidt, Aspen Mesh <granville@aspenmesh.io> (@gramidt)
-- Ken Haines, Microsoft <khaines@microsoft.com> (@khaines)
 - Neeraj Poddar, Aspen Mesh <neeraj@aspenmesh.io> (@nrjpoddar)
+- Niclas Schad, plusserver GmbH <Niclas.Schad@plusserver.com> (@ShuzZzle)
```
Comment (kd7lxl) [inline MAINTAINERS.md:3]:
```suggestion
- Niclas Schad, plusserver GmbH <Niclas.Schad@plusserver.com> (@ShuzZzle)
- Tom Hayward, Infoblox <thayward@infoblox.com> (@kd7lxl)
```

**Review by kd7lxl — APPROVED** [review]

File: MAINTAINERS.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
...
+- Tom Hayward, Infoblox <thayward@infoblox.com> (@kd7lxl)
```
Comment (kd7lxl) [inline MAINTAINERS.md:4]: Hmm, mentions only work without parenthesis I guess.
```suggestion
- Niclas Schad, plusserver GmbH <Niclas.Schad@plusserver.com> ( @ShuzZzle )
- Tom Hayward, Infoblox <thayward@infoblox.com> ( @kd7lxl )
```

Comment (ShuzZzle) [reply MAINTAINERS.md:4]: I don't think they are supposed to render.
See: https://github.com/github/markup/issues/209

Comment (ShuzZzle) [reply MAINTAINERS.md:4]: We could create a Link with the mention as the label but I don't really care

**Review by kd7lxl — APPROVED** [review]

**Pull request merged**